docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -1,486 +1,488 @@
-  running the Proxy/Auth Services.
+- Access to a running Teleport cluster, `tctl` admin tool, and `tsh` client tool 
+  version >= (=teleport.version=). 
+  
+  You can verify the tools you have installed by running the following commands:
 
-</Admonition>
+  ```code
+  $ tctl version
+  # Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
 
-## Prerequisites
+  You can download these tools by following the appropriate [Installation 
+  instructions](../installation.mdx#installation-instructions) for the Teleport 
+  edition you use.
 
-This guide requires you to have:
+- A Linux server to run the Teleport Windows Desktop Service.
+  You can use an existing server that runs the Teleport agent for other resources. 
+  
+- An Active Directory domain that is configured for LDAPS. Because Teleport requires an
+  encrypted LDAP connection, you should verify that your domain uses Active Directory 
+  Certificate Services or a non-Microsoft certification authority (CA) to issue LDAPS 
+  certificates.
 
-- An Active Directory domain, configured for LDAPS (Teleport requires an
-  encrypted LDAP connection). Typically this means installing
-  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+  If you don't have Active Directory or Active Directory Certificate Services installed, 
+  you can add roles and services using Server Manager or refer to [Configure access for 
+  Active Directory with scripts](./active-directory.mdx) for how to
+  use the Teleport-provided scripts to install these services.
 
-<Admonition title="Azure AD" type="warning">
+- Administrative access to a domain controller.
 
-Microsoft's Azure Active Directory (Azure AD) offering does not support the
-Kerberos authentication protocol, which is required for the certificate-based
-authentication described in this section.
+- (!docs/pages/includes/tctl.mdx!)
 
-At this time, Teleport does not support integration with Azure AD, however
-Teleport Enterprise customers can access Windows desktops (including those
-joined to Azure AD) using local accounts via the process described in [Getting
-Started with Desktop Access](./getting-started.mdx).
-</Admonition>
+## Step 1/7. Create a restrictive service account
 
-- Access to a Domain Controller
+Teleport requires a service account to connect to your Active Directory domain.
+You should create a dedicated service account with restrictive permissions
+for maximum security.
 
-(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+To create the service account:
 
-- A Linux server to run the Teleport Desktop Service on.
+1. Open PowerShell on a Windows domain computer.
 
-  You can reuse an existing server running any other Teleport instance.
+1. Create a service account with a randomly-generated password by copying and pasting 
+   the following script into the PowerShell console: 
+   
+   ```powershell
```

Comment:
The syntax highlighting looks off in the vercel preview, I wonder if `powershell` is a supported syntax.

Reply:
I'm not sure what it should look like. @ptgott maybe you know about the syntax highlighting for powershell?

Reply:
`powershell` is in the list of supported syntax labels for our highlighting library: https://github.com/wooorm/lowlight#common

You might want to try an alias (`pwsh`, `ps`, or `ps1`) just in case. 

Maybe `code` would also work here if we were to add `$` prompts to each command line so the docs engine renders them as it does other copy-pasteable commands?